pkg/rke2/kp.go

```diff
@@ -0,0 +1,52 @@
+package rke2
+
+import (
+	"context"
+
+	"github.com/rancher/k3s/pkg/cli/cmds"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/k3s-io/helm-controller/pkg/generated/controllers/helm.cattle.io"
+
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I feel like gofmt would tidy this a bit?

Reply:
Yeah, GoLand does that sometimes and I didn't notice.